API.md

```diff
@@ -74,6 +74,30 @@ These changes address the two most common requests; "how do I filter on `X`?" an
 
 **This change also allows user to leverage *any* existing transform or write stream in the node ecosystem to be used with good.**
 
+### Reporter Type
```

Comment:
I think we're mixing two things together... to ME a reporter is the pipeline we create and the things composing the pipeline are streams. We should decided on a consistent name and adjust accordingly.